src/components/panel/panel.js

```diff
@@ -489,6 +482,16 @@ angular
 
 /**
  * @ngdoc method
+ * @name MdPanelPosition#tryPanelPositions
```

Comment:
We can, I just thought it would be confusing to have two different names for the same functionality.

The duplication doesn't bother me, since this is a manual fallback. Another part of #7878 is to try to do the axis-flipping automatically